install_config/topics/ocp_osp_install.adoc

```diff
@@ -10,3 +10,11 @@ Run the installer playbook to install Red Hat OpenShift Container Platform:
 ----
 $ ansible-playbook /usr/share/ansible/openshift-ansible/playbooks/openstack/openshift-cluster/install.yml
 ----
+
+[NOTE]
+====
+{product-title} is supported on the last Red Hat OpenStack Platform, as
+well as on the Long Term Support Red Hat OpenStack Platfrom.
+So, at this time OSP 14 and the LTS version (OSP 13).
```

Comment:
This note/callout needs to be managed differently between 3.10 and 3.11. 
3.10 only runs (or is supported) on OSP 13 (if I remember the matrix correctly).

Reply:
This documentation is for 3.11, right?

Reply:
Yes but we will need 3.10 pr's too.

Reply:
done